app/src/main/res/values-b+nb/strings.xml

```diff
@@ -129,18 +129,18 @@
     <string name="car_trim_plus">Plus/Luxury</string>
     <string name="car_trim_exclusive">Exclusive</string>
     <string name="car_trim_ultimate">Ultimate</string>
-    <string name="car_unknown">Unknown</string>
+    <string name="car_unknown">Ukjent</string>
 
-    <string name="car_south_korea">Syd Korea</string>
+    <string name="car_south_korea">Sør-Korea</string>
 
     <!-- Background info for SOH display -->
-    <string name="calc_soh_source_pid_unknown">Still unknown for car model year</string>
-    <string name="calc_soh_source_awaiting">Awaiting data</string> // TODO
-    <string name="calc_soh_source_bms_not_read">Awaiting read of BMS data</string>  //TODO
-    <string name="calc_soh_source_deterioration_bms_not_set">BMS has no deterioration value yet</string> //TODO
-    <string name="calc_soh_source_deterioration">Calculated from BMS deterioration min/max</string> //TODO
-    <string name="calc_soh_source_pid_scaled_bms_not_set">BMS has no SOH value yet</string> //TODO
-    <string name="calc_soh_source_pid_scaled">BMS SOH value</string>
+    <string name="calc_soh_source_pid_unknown">Fortsatt ukjent for bilmodellåret</string>
+    <string name="calc_soh_source_awaiting">Venter på data</string>
+    <string name="calc_soh_source_bms_not_read">Venter på lesing av BMS-data</string>
+    <string name="calc_soh_source_deterioration_bms_not_set">BMS har ingen forringelsesverdi enda</string>
```

Comment:
Det er en teknisk term, "deterioration", så jeg vil foretrække at den bibeholdes

Reply:
Ta en kikk på mine siste to commits, ser det greit ut nå?